keyboards/handwired/ferris/info.json

```diff
@@ -0,0 +1,183 @@
+        },
+        {
+          "label": "I",
+          "x": 9,
+          "y": 0
+        },
+        {
+          "label": "R",
+          "x": 3,
+          "y": 0.28
+        },
+        {
+          "label": "U",
+          "x": 8,
+          "y": 0.28
+        },
+        {
+          "label": "W",
+          "x": 1,
+          "y": 0.31000000000000005
+        },
+        {
+          "label": "O",
+          "x": 10,
+          "y": 0.31000000000000005
+        },
+        {
+          "label": "T",
+          "x": 4,
+          "y": 0.42000000000000004
+        },
+        {
+          "label": "Y",
+          "x": 7,
+          "y": 0.42000000000000004
+        },
+        {
+          "label": "Q",
+          "x": 0,
+          "y": 0.93
+        },
+        {
+          "label": "P",
+          "x": 11,
+          "y": 0.93
+        },
+        {
+          "label": "D",
+          "x": 2,
+          "y": 1
+        },
+        {
+          "label": "K",
+          "x": 9,
+          "y": 1
+        },
+        {
+          "label": "F",
+          "x": 3,
+          "y": 1.28
+        },
+        {
+          "label": "J",
+          "x": 8,
+          "y": 1.28
+        },
+        {
+          "label": "S",
+          "x": 1,
+          "y": 1.31
+        },
+        {
+          "label": "L",
+          "x": 10,
+          "y": 1.31
+        },
+        {
+          "label": "G",
+          "x": 4,
+          "y": 1.42
+        },
+        {
+          "label": "H",
+          "x": 7,
+          "y": 1.42
+        },
+        {
+          "label": "A",
+          "x": 0,
+          "y": 1.93
+        },
+        {
+          "label": ":",
+          "x": 11,
+          "y": 1.93
+        },
+        {
+          "label": "C",
+          "x": 2,
+          "y": 2
+        },
+        {
+          "label": "<",
+          "x": 9,
+          "y": 2
+        },
+        {
+          "label": "V",
+          "x": 3,
+          "y": 2.28
+        },
+        {
+          "label": "M",
+          "x": 8,
+          "y": 2.28
+        },
+        {
+          "label": "X",
+          "x": 1,
+          "y": 2.31
+        },
+        {
+          "label": ">",
+          "x": 10,
+          "y": 2.31
+        },
+        {
+          "label": "B",
+          "x": 4,
+          "y": 2.42
+        },
+        {
+          "label": "N",
+          "x": 7,
+          "y": 2.42
+        },
+        {
+          "label": "Z",
+          "x": 0,
+          "y": 2.93
+        },
+        {
+          "label": "?",
+          "x": 11,
+          "y": 2.93
+        },
+        {
+          "label": "⌫",
+          "x": 3.5,
+          "y": 3.75
+        },
+        {
+          "label": "⏎",
+          "x": 7.5,
+          "y": 3.75
+        },
+        {
+          "label": "⎵",
+          "x": 4.5,
+          "y": 4
+        },
+        {
+          "label": "⎵",
+          "x": 6.5,
+          "y": 4
+        }
+      ]
+    }
+  }
+}
```

Comment:
```suggestion
{
    "keyboard_name": "Ferris",
    "url": "https://github.com/pierrechevalier83/ferris/",
    "maintainer": "@pierrec83",
    "width": 12,
    "height": 4.75,
    "layouts": {
        "LAYOUT": {
            "layout": [
                {"x": 0, "y": 0.93},
                {"x": 1, "y": 0.31},
                {"x": 2, "y": 0},
                {"x": 3, "y": 0.28},
                {"x": 4, "y": 0.42},

                {"x": 7, "y": 0.42},
                {"x": 8, "y": 0.28},
                {"x": 9, "y": 0},
                {"x": 10, "y": 0.31},
                {"x": 11, "y": 0.93},

                {"x": 0, "y": 1.93},
                {"x": 1, "y": 1.31},
                {"x": 2, "y": 1},
                {"x": 3, "y": 1.28},
                {"x": 4, "y": 1.42},

                {"x": 7, "y": 1.42},
                {"x": 8, "y": 1.28},
                {"x": 9, "y": 1},
                {"x": 10, "y": 1.31},
                {"x": 11, "y": 1.93},

                {"x": 0, "y": 2.93},
                {"x": 1, "y": 2.31},
                {"x": 2, "y": 2},
                {"x": 3, "y": 2.28},
                {"x": 4, "y": 2.42},

                {"x": 7, "y": 2.42},
                {"x": 8, "y": 2.28},
                {"x": 9, "y": 2},
                {"x": 10, "y": 2.31},
                {"x": 11, "y": 2.93},

                {"x": 3.5, "y": 3.75},
                {"x": 4.5, "y": 4},

                {"x": 6.5, "y": 4},
                {"x": 7.5, "y": 3.75}
            ]
        }
    }
}
```
The order of the keys was wrong, and there were still some rounding errors from rotation. You can preview an info.json in the Configurator by pressing Ctrl+Shift+I, and turn on fast input to check the order.